apps/app/src/pages/Owner/GeneralSettings.js

```diff
@@ -42,7 +56,7 @@ const FeatureIcon = (props) => <Icon mt="-1px" {...props} />;
 const PlanName = (props) => <x.div fontSize="4xl" mb={3} {...props} />;
 
 const Plan = ({ purchase }) => {
-  if (!purchase.plan) return <PlanName mb={2}>No plan</PlanName>;
+  if (!purchase?.plan) return <PlanName mb={2}>No plan</PlanName>;
```

Comment:
No plan does not exist. It fallbacks on free plan. If not plan is defined in database then we have a special case where we say that there is no plan configured on this Argos instance.